pl/mozorg/contribute/index.lang

```diff
@@ -82,123 +82,123 @@ is about more than lending a hand. It’s about learning, collaborating and sha
 
 
 # Caption used under Mozillians photo. You can replace $middot; with a different separator if it doesn't work for your locale
 ;Spain &middot; Activism
-Spain &middot; Activism
+Hiszpania &middot; Aktywizm
 
 
 # Caption used under Mozillians photo. You can replace $middot; with a different separator if it doesn't work for your locale
 ;Philippines &middot; Activism
-Philippines &middot; Activism
+Filipiny &middot; Aktywizm
 
 
 # Caption used under Mozillians photo. You can replace $middot; with a different separator if it doesn't work for your locale
 ;Switzerland &middot; Coding
-Switzerland &middot; Coding
+Szwajcaria &middot; Programowanie
 
 
 # Caption used under Mozillians photo. You can replace $middot; with a different separator if it doesn't work for your locale
 ;India &middot; Teaching
-India &middot; Teaching
+Indie &middot; Nauczanie
 
 
 ;Read his story
-Read his story
+Poznaj jego historię
 
 
 ;Read her story
-Read her story
+Poznaj jej historię
 
 
 ;Want to get involved? Here’s how.
-Want to get involved? Here’s how.
+Chcesz nam pomóc? Tutaj dowiesz się jak.
 
 
 ;Choose what interests you and tell us how you’d like to help
-Choose what interests you and tell us how you’d like to help
+Wybierz czym się interesujesz i powiedz, jak chciał(a)byś pomóc
 
 
 ;Stand by for an email that will connect you to the Mozilla community
-Stand by for an email that will connect you to the Mozilla community
+Oczekuj maila, który połączy cię ze społecznością Mozilli
 
 
 ;Create your Mozillians account and share your contributions with the world
-Create your Mozillians account and share your contributions with the world
+Utwórz swoje konto i podziel się swoim wkładem z całym światem
 
 
 ;Get started
-Get started
+Zapoznaj się
 
 
 ;Not ready to dive in just yet?
-Not ready to dive in just yet?
+Nie jesteś jeszcze gotowy?
 
 
 ;Here are a few other ways you can support Mozilla right now.
-Here are a few other ways you can support Mozilla right now.
+Dowiedz się, jak jeszcze możesz pomóc Mozilli.
 
 
 ;Donate to Mozilla
-Donate to Mozilla
+Przekaż dotację
 
 
 ;Read our Manifesto
-Read our Manifesto
+Przeczytaj nasz manifest
 
 
 ;Like us on Facebook
-Like us on Facebook
+Polub nas na Facebooku
 
 
 ;Follow us on Twitter
-Follow us on Twitter
+Obserwuj nas na Twitterze
 
 
 ;Upcoming event
-Upcoming event
+Nadchodzące wydarzenie
 
 
 ;See all events
-See all events
+Zobacz wszystkie wydarzenia
 
 
 ;Communicate
-Communicate
+Komunikuj się
 
 
 ;Mozilla communities
-Mozilla communities
+Społeczności Mozilli
 
 
 ;Mailing lists
-Mailing lists
+Listy mailingowe
 
 
 ;IRC channels
-IRC channels
+Kanały IRC
 
 
 ;More links
-More links
+Więcej linków
 
 
 ;Submit Firefox feedback
-Submit Firefox feedback
+Prześlij opinię o Firefoksie
 
 
 ;Explore Mozilla careers
-Explore Mozilla careers
+Zobacz oferty pracy w Mozilli
 
 
 ;Get help with Firefox
-Get help with Firefox
+Zdobąd pomoc dotyczącą Firefoksa
```

Comment:
Literówka w „Zdobąd”, ale to nie ma znaczenia, bo tutaj lepiej by było dać „Pomoc dla Firefoksa” lub coś podobnego.